Wireframe/index.html

```diff
@@ -4,10 +4,43 @@
     <meta charset="UTF-8">
```

Comment:
I don't think any of this code belongs in here. You need to open a new PR with the wireframe work in it, please. Come to class to get help with this if you are blocked.